utils.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/pkg/errors"
+)
+
+func findManifest() (*model.Manifest, error) {
+	_, manifestFilePath, err := model.FindManifest(".")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to find manifest in current working directory")
+	}
+	manifestFile, err := os.Open(manifestFilePath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to open %s", manifestFilePath)
+	}
+	defer manifestFile.Close()
+
+	var manifest model.Manifest
+	decoder := json.NewDecoder(manifestFile)
+	if err = decoder.Decode(&manifest); err != nil {
+		return nil, errors.Wrap(err, "failed to parse manifest")
```

Comment:
for future iterations could be interesting to require some struct fields to be not empty after decoding the JSON

Reply:
I mean, we want to use the manifest to check whether the plugin has webapp/server or not. But the manifest itself could be validated too for points like:
- empty plugin id,name,description,version,min_server_version
- check if min_server_version is valid
- check URLs are valid
- not webapp nor server is enabled
- check nested settings (same, non-empty required fields)

Reply:
not saying that this should be covered in this PR tho

Reply:
Made a ticket here https://github.com/mickmister/mmplug/issues/2